src/runtime/symbols.ts

```diff
@@ -8,5 +8,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-// tslint:disable-next-line: variable-name
-export const Symbols = {identifier: Symbol('id')};
+// TypeScript seems to lose the necessary type info if this symbol is wrapped in an object and then
+// used as an interface key (e.g. 'interface Foo { [Symbols.internals]: {...} }'), so we just have
+// to export it as a standard variable.
+export const SYMBOL_INTERNALS = Symbol('internals');
```

Comment:
What would you think about defining separate symbols for entities and handles? I feel like this is a little more obvious:

```
entity[ENTITY_INTERNALS].whatever();
```

Pretty much equivalent though..

Reply:
Handles don't actually have an internals object. References do, but they necessarily need to use the same key (because they mimic Entities from the Handle point of view).

That said, I'm fine with still renaming the symbol. WDYT?

Reply:
Oh right, that makes sense. The name's probably fine then as it is.